### PR TITLE
Add twitter link.

### PR DESCRIPTION
**Pull request by biocom**
This adds a Twitter icon/link in the header.

Screen Shot 2021-05-20 at 18 25 31
Screen Shot 2021-05-20 at 18 25 45

### CODE REVIEW

**Review by anxolin — APPROVED** [review]

File: src/custom/components/Header/index.tsx [hunk 1]
```diff
@@ -127,6 +144,9 @@ export default function Header() {
           </AccountElement>
         </HeaderElement>
         <HeaderElementWrap>
+          <TwitterLink>
+            <a href="https://twitter.com/mevprotection" target="_blank" rel="noopener noreferrer" />
+          </TwitterLink>
```
Comment (alfetopito) [inline src/custom/components/Header/index.tsx:149]: Build failure:

screenshot_2021-05-20_11-01-53

**Review by alfetopito — CHANGES_REQUESTED** [review]

File: src/custom/components/Header/index.tsx (hunk 1 above)
Comment (biocom) [reply src/custom/components/Header/index.tsx]: Had a hunch my empty <a href> would fail :) My bad...fixed now!

**Review by alfetopito — APPROVED** [review]
:bird: :musical_note:

**Review by anxolin — APPROVED** [review]
Nice. As a nit pick, could we add the cursor animation on hover?

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia0.giphy.com%2Fmedia%2FSMKiEh9WDO6ze%2Fgiphy.gif&token=twitt" data-gitmeme-token="twitt"></a>

**Comment by biocom** [comment]
@anxolin Haha, sure, added :)

**Review by anxolin — APPROVED** [review]

**Review by W3stside — APPROVED** [review]

**Comment by anxolin** [comment]
@Mergifyio refresh

**Pull request merged**